addons/binding/org.openhab.binding.freemobilesms/README.md

```diff
@@ -0,0 +1,53 @@
+# FreeMobile SMS Binding
+
+The FreeMobile SMS binding povides support for sending SMS via [FreeMobile API](https://www.freenews.fr/freenews-edition-nationale-299/free-mobile-170/nouvelle-option-notifications-par-sms-chez-free-mobile-14817).
+
+## Supported Things
+
+This binding supports the following thing types:
+
+| Thing         | Thing Type | Description                                             |
+|---------------|------------|---------------------------------------------------------|
+| account       | Thing      | One callable account.                                   |
+
+## Discovery
```

Comment:
Just remove "example" sections where you cannot add useful content to, like this one.

Reply:
removed